app/views/assets/_usage_info_box.html.erb

```diff
@@ -22,6 +22,11 @@
       <p>
         <strong>Last updated</strong>: <%= date_as_string(resource.updated_at, true) -%>
       </p>
+      <% unless resource.updated_last_by.nil? || !current_user&.person&.member_of?(resource.projects)-%>
```

Comment:
instead, this would be better as

`if resource.updated_last_by && current_user&.person&.member_of?(resource.projects)`

avoiding double negatives

Reply:
`nil` can be treated equivalent as false in Ruby